src/Chip/index.tsx

```diff
@@ -32,46 +36,73 @@ const Chip = React.forwardRef(({
   onIconAfterClick,
   disabled,
   ...props
-}: IChip, ref: ForwardedRef<HTMLDivElement>) => (
-  <div
-    className={classNames(
-      CHIP_PGN_CLASS,
-      `pgn__chip-${variant}`,
-      className,
-      { disabled },
-    )}
-    ref={ref}
-    {...props}
-  >
-    {iconBefore && (
-      <div
-        className={classNames('pgn__chip__icon-before')}
-      >
-        <Icon src={iconBefore} />
-      </div>
-    )}
+}: IChip, ref: ForwardedRef<HTMLDivElement>) => {
+  const [isSelected, setIsSelected] = useState(false);
```

Comment:
Let's add `isSelected` prop instead of internal state

Reply:
Yeah, it`s good idea